src/server/models/list.ts

```diff
@@ -44,25 +44,21 @@ export async function findListByCountryAndType(country: CountryName, type: Servi
   }
 }
 
-export async function findListByAnnualReviewDate(annualReviewStartDate: Date, listItemStatuses: Status[] = [], isAnnualReview?: boolean): Promise<Result<List[]>> {
+export async function findListByAnnualReviewDate(annualReviewStartDate: Date, today: Date, listItemStatuses: Status[] = [], isAnnualReview?: boolean): Promise<Result<List[]>> {
```

Comment:
does today need to be passed in? won't it always be.... today?

Reply:
nope, not needed now.  Removed the date range as it should include all lists with an annual review start date prior to the argument passed in